src/index.html

```diff
@@ -9,10 +9,41 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+      <div>
+        <img src="images/imac.jpeg" alt="imacPh">
```

Comment:
The 'alt' attribute for the image is not very descriptive. Consider providing a more detailed description to improve accessibility, such as 'Image of an Apple A1419 iMac 27" Retina 5K Monoblock'.